lib/ansible/modules/net_tools/nmcli.py

```diff
@@ -1221,9 +1225,9 @@ def modify_connection_vxlan(self):
         elif self.ifname is not None:
             cmd.append(self.ifname)
         else:
-            cmd.append('vxlan%s' % self.vxlanid)
+            cmd.append('vxlan%s' % to_text(self.vxlan_id))
```

Comment:
Thanks this PR really helped me out!

These format statements don't need the to_text, as it will convert the variables to a string
https://docs.python.org/2/library/stdtypes.html#string-formatting

```
$ ipython
Python 3.8.2 (default, Feb 28 2020, 00:00:00) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.12.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: var = 10                                                                                                                                                                                        

In [2]: var                                                                                                                                                                                             
Out[2]: 10

In [3]: type(var)                                                                                                                                                                                       
Out[3]: int

In [4]: formatted = 'vlan%s' % var                                                                                                                                                                      

In [5]: formatted                                                                                                                                                                                       
Out[5]: 'vlan10'

In [6]: type(formatted)                                                                                                                                                                                 
Out[6]: str
```

Seems a shame to change everything to a String.
Seen a number of nice PRs for nmcli which haven't been merged and one impacts Fedora 31/32, RHEL8 and CentOS8.

Best of luck